src/components/erc20/marketplace/order_book.tsx

```diff
@@ -48,10 +48,11 @@ const orderToRow = (
     }, new BigNumber(0));
 
     const mySizeConverted = tokenAmountInUnits(mySize, baseToken.decimals, UI_DECIMALS_DISPLAYED_ORDER_SIZE);
+    const displayColor = !mySize.eq(new BigNumber(0)) ? undefined : '#dedede';
```

Comment:
Sorry for being a PITA, but since this expression is used twice and it's not exactly readable, can we extract it to a variable?

```
const isSizeEmpty = !mySize.eq(new BigNumber(0))
```

(or whichever name you prefer).

Reply:
Sure, no problem, it's done, please check it again